zio-http/src/main/scala-2/zhttp/http/IsRequest.scala

```diff
@@ -0,0 +1,16 @@
+package zhttp.http
+
+import scala.annotation.implicitNotFound
+import scala.language.implicitConversions
+
+@implicitNotFound(
+  "This operation is only allowed if the input is a type of Request. " +
+    "However your Http instance accepts the type ${A}, on which this operation can not be applied.",
```

Comment:
this will fail the scaladoc generation during publish, Please refer this issue https://github.com/scala/bug/issues/10333